reactor-client/src/main/java/com/aerospike/client/reactor/dto/KeyObject.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2012-2018 Aerospike, Inc.
+ *
+ * Portions may be licensed to Aerospike, Inc. under one or more contributor
+ * license agreements WHICH ARE COMPATIBLE WITH THE APACHE LICENSE, VERSION 2.0.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.aerospike.client.reactor.dto;
+
+import com.aerospike.client.Key;
+
+/**
+ * Container object for key identifier and data.
+ */
+public final class KeyObject {
+	/**
+	 * Unique identifier for record.
+	 */
+	public final Key key;
+
+	/**
+	 * Data.
+	 */
+	public final Object value;
+
+	/**
+	 * Initialize key and record.
+	 */
+	public KeyObject(Key key, Object value) {
+		this.key = key;
+		this.value = value;
+	}
+	
+	/**
+	 * Convert key and record to string.
+	 */
+	@Override
+	public String toString() {
```

Comment:
consider adding toString to other classes as well

Reply:
fixed